meta-mender-imx6ul/recipes-core/systemd/systemd-conf.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
+
+SRC_URI += "file://eth.network"
+
+FILES_${PN} += "${sysconfdir}/systemd/network/eth.network"
+
+do_install_append() {
+    install -d ${D}${sysconfdir}/systemd/network
+    install -m 0644 ${WORKDIR}/eth.network ${D}${sysconfdir}/systemd/network
+}
```

Comment:
This is not Mender related and should not be here. Actually there are similar definitions for this in [meta-mender-demo](https://github.com/mendersoftware/meta-mender/tree/master/meta-mender-demo), so this would just be redundant.